android/engine/src/main/java/org/smartregister/fhircore/engine/sync/SyncStrategy.kt

```diff
@@ -16,14 +16,44 @@
 
 package org.smartregister.fhircore.engine.sync
 
+import org.hl7.fhir.r4.model.Coding
+
 /**
  * SyncStrategy defines whether to sync resource based on the IDs of CareTeam, Location,
  * Organization and Practitioner. Each SyncStrategy represents a meta tag that is used by all synced
  * resource.
  */
-enum class SyncStrategy(val value: String) {
-  CARE_TEAM("CareTeam"),
-  LOCATION("Location"),
-  ORGANIZATION("Organization"),
-  PRACTITIONER("Practitioner")
+enum class SyncStrategy(val value: String, val tag: Coding) {
+  CARE_TEAM(
+    value = "CareTeam",
+    tag =
+      Coding().apply {
+        system = "https://smartregister.org/"
+        display = "Practitioner CareTeam"
```

Comment:
Is there a possibility for these codings to change? Do we want to configure them? Also can the SyncStrategy be part of `ApplicationConfiguration`? 

cc: @dubdabasoduba

Reply:
I think one thing from me is.

For the `system` to be more detailed.

Something like
`<Base Url>/practitioner-care-team`

Reply:
@ellykits Yes, we can configure them. I think that will give us more flexibility if we needed to change them in future. 

@FikriMilano mmh we can do that but then that URL doesn't exist. My thinking was since these are custom codes then redirecting to the SmartRegister website makes sense.

Reply:
Oh ok, so we will configure them and have that as it is for now.

We can easily change the `system` in the future if need be.